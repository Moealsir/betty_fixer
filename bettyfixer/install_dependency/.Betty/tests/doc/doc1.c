int op_add(int arg1, int arg2)
{
	return (arg1 + arg2);
}

int main(int ac, char **av)
{
	(void)ac;
	(void)av;
	op_add(1, 2);
	return (0);
}
