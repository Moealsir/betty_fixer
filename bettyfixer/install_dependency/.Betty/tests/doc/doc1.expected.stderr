tests/doc/doc1.c:1: warning: no description found for function op_add
tests/doc/doc1.c:6: warning: no description found for function main
